week-2/implement/lookup.js

```diff
@@ -1,17 +1,32 @@
-function createLookup() {
+function createLookup(countryCurrencyPairs) {
   // implementation here
+  const obj ={};
+  for(const pair of countryCurrencyPairs){
+    if(pair.length ===2){
+      const [countryCode, currencyCode]= pair;
+
+      obj[countryCode]=currencyCode;
+
+    }else{
+      const [countryCode, currencyCode] = countryCurrencyPairs;
+      obj[countryCode] = value;
```

Comment:
This code needs a bit of a tidy I'm afraid. I think you only really need the firstpart of the if - the else section should return an error